### DISCUSSION TITLE
Ability to add content-items in list/sort view

### DISCUSSION THREAD

**Discussion by renderfarmer** [2sic/2sxc#2478] [Ideas]
It would be great if there would be an option to add content-items in the list/sort view and be able to re-organize items after adding without the dialog/modal being closed or the page reloaded. In some use cases this would be a very comfortable content management feature and speed up the editing workflow.

**Comment by iJungleboy** [comment]
Could you add some screenshots for clarity?

**Reply by renderfarmer** [reply]
Basically like [this »](https://d.pr/i/BX5rop)

So I can add a new item after each existing item or I can remove items, all from right within the list/sort view without the page being reloaded while editing.

**Reply by iJungleboy** [reply]
the picture of @renderfarmer here for easier use (BTW @renderfarmer you can paste images directly into github)

![image](https://user-images.githubusercontent.com/5321186/129903909-6f8546b4-49a8-4c83-86cd-eb5c5a6256b8.png)

**Reply by iJungleboy** [reply]
**Remove** would make sense I guess. 

The implementation for **add** would probably be a bit simpler - like having an "add more" section below similar to entity-management in the normal entity picker. Since you can then sort it where you need it, it would still be simple. 

I'm not sure if "add dummy" would be relevant here - do you think it is?

Make sense? would this solve your need?

**Reply by renderfarmer** [reply]
No, "add dummy" wouldn't be that useful, but add existing definitely. So, AddNew + AddExisting + Remove 👌🏼

**Reply by renderfarmer** [reply]
Are you planning to look at this anytime soon? We always running into this... would be very useful. Thanks 🙏🏼

**Reply by iJungleboy** [reply]
Hi @renderfarmer - I'm just back from a mission in Tanzania doing https://school-sys.org/en/ - that's why you haven't heard from me.

It's not on our priorities ATM, even though it would make sense to do it. 

If you want us to prioritize it, please consider sponsoring it - ca. 2h of development should be enough for remove only, or ca. 4h for add new, add existing + remove

**Reply by renderfarmer** [reply]
Wow that's great. Going to Africa never disappoints 😃... Great project!

So 4h for all buttons... how much would that be? Thanks 🙏🏼

**Reply by iJungleboy** [reply]
Small things are 210 USD or ca. 190 EUR per hour. 
Rebates kick in if it's a lot of work, like 5 days or more, so that doesn't apply here ;)
Details https://2sxc.org/en/contact

**Reply by renderfarmer** [reply]
Ok, would like to take your offer now to have these buttons integrated.
